roles/openshift_master_facts/tasks/main.yml

```diff
@@ -74,7 +74,7 @@
       ldap_ca: "{{ openshift_master_ldap_ca | default(lookup('file', openshift_master_ldap_ca_file) if openshift_master_ldap_ca_file is defined else None) }}"
       openid_ca: "{{ openshift_master_openid_ca | default(lookup('file', openshift_master_openid_ca_file) if openshift_master_openid_ca_file is defined else None) }}"
       request_header_ca: "{{ openshift_master_request_header_ca | default(lookup('file', openshift_master_request_header_ca_file) if openshift_master_request_header_ca_file is defined else None) }}"
-      registry_url: "{{ oreg_url | default(None) }}"
+      registry_url: "{{ oreg_url | default(oreg_url_master) | default(None) }}"
```

Comment:
This would mean that if oreg_url and oreg_url_master are both set then you'll get oreg_url value. To me I'd expect this to work in the opposite manner, the more specific value should be used. What do you think?